src/network/http.js

```diff
@@ -0,0 +1,30 @@
+class http {
+  constructor(config) {
+    this.baseUrl = `http://${config.hostname}:${config.port}`;
+    this.header = {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    };
+  }
```

Comment:
1. For the same reasons mention in the AMQP class, this should be `HTTP`
2. The header will now have a variable `Authorization` field with the `authToken`. For this reason, the header should no longer be in the constructor, but instead it should be build within the context of the request. 

```suggestion
class HTTP {
  constructor(config) {
    this.baseUrl = `http://${config.hostname}:${config.port}`;
  }
```